srcpkgs/llvm/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'llvm'
 pkgname=llvm
-version=12.0.0
+version=15.0.6
 revision=1
 build_style=meta
-depends="llvm12"
-replaces="llvm11>=0 llvm10>=0 llvm9>=0 llvm8>=0 llvm7>=0 llvm6.0>=0 llvm3.9>=0"
+depends="llvm15"
 short_desc="LLVM meta package"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="Public Domain"
 homepage="https://www.voidlinux.org"
+replaces="llvm12>=0 llvm11>=0 llvm10>=0 llvm9>=0 llvm8>=0 llvm7>=0 llvm6.0>=0 llvm3.9>=0"
```

Comment:
llvm12 currently provides clang et al, which should be removed:
https://github.com/void-linux/void-packages/commit/0d9f8330b556561ee38de7d6a0d4280f56cdd58e

Also, you need to rebuild/update everything depending on llvm.

Here is the llvm12 update for reference:
https://github.com/void-linux/void-packages/commits/e8c2d0e32888730ad5fa16e1441de1a4e1a3a64a

Ofc a bunch of stuff was updated after that (i.e. rust, ldc, etc) but just for reference.